### ISSUE TITLE
Some question on discord-bot

### ISSUE THREAD

**Issue by carlcc** [fiatrete/OpenDAN-Personal-AI-OS#20]
1. According to `README.md` of `agent_jarvis` and `discord_bot`. There are conflicts in their listening port (both are 10000).
2. I cannot find a full guide to deploy the discord bot, jarvis and discord-bot are both server, how can they communication to each other?

**Comment by Synthintel0** [comment]
Do you have a corresponding screenshot?

**Comment by Synthintel0** [comment]
I tried to go through the process of deploying Jarvis and the discord bot, and indeed encountered the issue you mentioned.
And I figured out what the problem was.
When Jarvis works with discord bot, Jarvis used as client.
We  need to set `JARVIS_SERVER_MODE=false` and `JARVIS_BOT_SERVER_URL` need to be set.
For example: 
- Suppose Jarvis and the discord bot are deployed on the same server.
- If discord bot set `WEBSOCKET_PORT=10000`, then We must set `JARVIS_BOT_SERVER_URL = http://localhost:10000` and `JARVIS_SERVER_MODE=false`.
